cyw43-pio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cyw43-pio"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
Shouldn't this be 0.7? If someone is using `embassy-rp = 0.7` and `cyw43-pio = 0.6`, they will be auto-updated to this version and will need to update embassy-rp, too.